engines/common/nucleus/particles/assets.html.twig

```diff
@@ -25,7 +26,36 @@
                 {% endfor %}
             {% endfor %}
         {% endif %}
+        
+        <script src="{{ url(script.location) }}" type="text/javascript"{{ attr_extra|raw }}></script>
+        
+        {% if script.inline %}
+            <script type="text/javascript">{{ script.inline }}</script>
+        {% endif %}
+        
+    {% endif %}
+    {% endfor %}
+{% endblock %}
 
+{% block javascript_footer %}
+    {% for script in particle.javascript %}
+    
+    {% if script.position == true %}
+        {% set attr_extra = '' %}
+        {% if script.extra %}
+            {% for attributes in script.extra %}
+                {% for key, value in attributes %}
+                    {% set attr_extra = attr_extra ~ ' ' ~ key|e ~ '="' ~ value|e('html_attr') ~ '"' %}
+                {% endfor %}
+            {% endfor %}
+        {% endif %}
+        
         <script src="{{ url(script.location) }}" type="text/javascript"{{ attr_extra|raw }}></script>
```

Comment:
Does the code, as is, just insert a link to an external script file at the end of the body tag? Or does it inline the script?

Is there any benefit to inlining the script over just linking to the script other than reducing http request(s)?

Best regards,

Ali

> On 15 Jul 2015, at 22:05, Matias Griese <notifications@github.com> wrote:
> 
> In engines/common/nucleus/particles/assets.html.twig:
> 
> >  
> > +{% block javascript_footer %}
> > +    {% for script in particle.javascript %}
> > +    
> > +    {% if script.position == true %}
> > +        {% set attr_extra = '' %}
> > +        {% if script.extra %}
> > +            {% for attributes in script.extra %}
> > +                {% for key, value in attributes %}
> > +                    {% set attr_extra = attr_extra ~ ' ' ~ key|e ~ '="' ~ value|e('html_attr') ~ '"' %}
> > +                {% endfor %}
> > +            {% endfor %}
> > +        {% endif %}
> > +        
> >          <script src="{{ url(script.location) }}" type="text/javascript"{{ attr_extra|raw }}></script>
> This line should be in {% else %} of the next if.
> 
> —
> Reply to this email directly or view it on GitHub.
>